.env.example

```diff
@@ -51,6 +51,36 @@ INVITATION_CODE=try-huginn
 # If you don't want to require new users to have an invitation code in order to sign up, set this to true.
 SKIP_INVITATION_CODE=false
 
+# If you require your users to confirm their email address after sign up set this to false.
```

Comment:
Minor grammer:

`If you'd like to require new users to confirm their email address after sign up, set this to false.`